src/main-app.js

```diff
@@ -174,6 +260,42 @@ export class MainApplication extends LitElement {
     }
   }
 
+  async _startScan() {
+    const scanOption = { recordType: "mime" };
+    try {
+      if (this._btnScanSvg.classList.contains("disabled-scan")) {
+        await this._reader.scan(scanOption);
+        this._btnScanSvg.classList.remove("disabled-scan");
+        this._snackbar.labelText = "Add item or touch an NFC tag!";
+        this._actionBtn.textContent = "";
+        this._snackbar.open();
+        if (this._firstScan) {
+          this._hideScanBg = true;
+          this._firstScan = false;
```

Comment:
why two veriables, don't they mean the same?

Reply:
Now only one is sufficient. Done.